sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/webpack-dev-server/test

```diff
@@ -1,13 +1,13 @@
 # the server should properly start
 > fastOptJS::startWebpackDevServer
-> html index.html 20000
+> html index.html 20000 true
 
 # the server should properly stop
 > fastOptJS::stopWebpackDevServer
--> html index.html
+> html index.html 1000 false
```

Comment:
It doesn’t work if you remove the `shouldPass` parameter and prefix this line with a `-`?

Reply:
i think this was incorrect in the original. if you check https://github.com/scalacenter/scalajs-bundler/blob/master/sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/webpack-dev-server/build.sbt#L27 you'll see that with only one parameter, it would fail before even trying to run webpack. My idea was to always get to the place where the html tests are really executed

Reply:
Good point, we should have added the timeout value.